src/components/atoms/AddComponentButton.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Card, Text } from 'evergreen-ui';
+import { dashboardItemsVar } from '../../cache/cache';
+import { WEATHER_MET_API } from '../../queries/metApi';
+
+const AddComponentButton: React.FC = () => {
+    const addDashboardItem = () => {
+        dashboardItemsVar([...dashboardItemsVar(), { size: 3, withText: false, query: WEATHER_MET_API }]);
+    };
```

Comment:
This causes the storybook to crash when you try to click the button. You should either mock it in some way or provide a provider in the storybook example.